### PR TITLE
Cg 103 people connect to api

### PR DESCRIPTION
**Pull request by jennikate**

### CODE REVIEW

File: src/components/Forms/FormPeople.jsx [hunk 1]
```diff
@@ -1,65 +1,145 @@
...
+      axios.post(`${apiPath}/user/people`, getFieldsToSubmit(), {
+        headers: { Authorization: `Bearer ${Auth.retrieveToken()}` },
+      })
+        .then(() => {
+          history.goBack(); // Return to page you came from
+          clearFormData();
+        })
+        .catch((err) => {
+          if (err.response) {
+            if (err.response) {
```
Comment (joshuadiablito) [inline src/components/Forms/FormPeople.jsx]: double `err.response`

File: src/components/Forms/FormPeople.jsx [hunk 2]
```diff
@@ -77,9 +157,9 @@ const FormPeople = (props) => {
           <div className="govuk-grid-column-two-thirds">
             <h1 className="govuk-heading-xl">Save a person</h1>
             <p className="govuk-body-l">Provide the details of the person you want to add to your list of saved people.</p>
-            <form>
+            <form id="CreatePerson">
```
Comment (joshuadiablito) [inline src/components/Forms/FormPeople.jsx]: any reason for using an id?

**Comment by joshuadiablito** [comment]
some general points:
- localstorage - would try and avoid its usage, forms can be temporary, anything more permanent will exist after interacting with the APIs (creating a voyage, returning list of people, etc)
- remove moment, use date-fns
- classnames (in the future!)
- Reduce repetition? There is a lot of repetition for the displaying of errors, you could create a component for it
- validation is quite complicated - could be prone to errors itself due to complex nature
- rule property of validationRules doesn’t appear to be used

File: src/components/Forms/FormPeople.jsx (hunk 2 above)
Comment (jennikate) [reply src/components/Forms/FormPeople.jsx]: Lets me take the users back to the top of the form if there are errors (there's a page level error box as well as individual fields)

File: src/components/Forms/FormPeople.jsx (hunk 1 above)
Comment (jennikate) [reply src/components/Forms/FormPeople.jsx]: fixed

Comment (jennikate) [reply src/components/Forms/FormPeople.jsx]: will push in next branch

**Pull request merged**